data/wanderer/wanderers middle.txt

```diff
@@ -2851,8 +2851,20 @@ mission "Wanderers: Sestor Alt: Alnilam 2"
 		government "Navy (Oathkeeper)"
 		personality escort
 		ship "Cruiser (Mark II)" "R.N.S. Dauntless"
+		ship "Combat Drone" "Alpha One" # them being Alpha is intentional
```

Comment:
It being intentional is all well and good, but for what reason?
This comment doesn't tell me that. It might encourage a reader to investigate further, which would bring them to this PR, which also doesn't explain it.
In general, if a ship has a specific name given to it, it should be assumed that that name was chosen for a reason anyway. Otherwise, a random name would be used. So, this comment doesn't really provide any information at all.
I would suggest that, if you think it is noteworthy, it should be explained in the PR description; we don't document every choice of word in the data files.